server/server.js

```diff
@@ -193,6 +198,10 @@ export function expressEndpoint({ uri = '/', logger = defaultLogger, enableCors
             return res.status(200).send();
         }
 
+        if (req.method.toLowerCase() === HTTP_METHOD.POST && !req.body) {
+            return res.status(400).send(req);
```

Comment:
I think `.send()` takes a message right? Like `.send('Expected post body')`

Reply:
Good catch!  Don't even remember doing that.  Intention was to do the above for 200.